actions/post-preview-urls/main.test.ts

```diff
@@ -0,0 +1,273 @@
+import {postPreviewUrls} from './main'
+import * as utils from '../utils'
+import * as github from '@actions/github'
+
+jest.mock('../utils')
+jest.mock('@actions/core')
+jest.mock('@actions/github')
+
+// just making sure the mock methods are correctly typed
+const mockedUtils = utils as jest.Mocked<typeof utils>
+const mockedGithub = github as jest.Mocked<typeof github>
+
+// reset the counter on mock fn calls after every test
+afterEach(() => jest.clearAllMocks())
```

Comment:
Usually a best practice to do that in a `beforeEach()` hook, just to make sure that the code runs even if the previous test crashed in a very bad way.